src/sagemaker/chainer/model.py

```diff
@@ -54,14 +59,17 @@ class ChainerModel(FrameworkModel):
 
     __framework_name__ = "chainer"
 
+    LATEST_VERSION = "5.0.0"
```

Comment:
should we instead read this from the estimator so that we don't have to update multiple constants?

Reply:
Yes. Duplicating the constant increases future efforts of maintaining them. 

However, Model class does not import corresponding estimator and LATEST_VERSION is an instance variable of an estimator object.  We need to do:

`import sagemaker.chainer.estimator as estimator`
and then
`estimator.LATEST_VERSION`

Model classes currently do not import corresponding Estimator classes, I was wondering if such imports worth it for "LATEST_VERSION"

Reply:
hmm...maybe it is worth moving it to `defaults.py` then even though it's not a default.

Reply:
Can we add a LATEST_VERSION in ``defaults.py`` but still keep the one in ``Estimator`` class? so that customer code which uses estimator.LATEST_VERSION won't break

Reply:
so you'll still have to set it in estimator, but you can do something like

```python
from sagemaker.<framework> import defaults

class <Framework>:
    LATEST_VERSION = defaults.LATEST_VERSION
```